### DISCUSSION TITLE
"Advancing Accessibility: Empowering Users and Developers in GitHub's Ecosystem"

### DISCUSSION THREAD

**Discussion by Theternos** [community/community#110328] [Accessibility]
### Select Topic Area

Question

### Body

Certainly! Here's a discussion topic focusing on the accessibility of GitHub and GitHub products:

"In what ways can GitHub further enhance accessibility features within its platform to ensure equitable access for users with disabilities, and how can developers contribute to making open-source projects hosted on GitHub more accessible to all?"

**Comment by sanjay-ar** [comment]
Comprehensive Accessibility Guidelines: GitHub should establish and adhere to comprehensive accessibility guidelines, ensuring that all aspects of the platform, including the user interface, navigation, and features, are accessible to individuals with disabilities. This includes implementing proper keyboard navigation, providing alternative text for images, and ensuring compatibility with screen readers.

**Reply by Theternos** [reply]
ok

**Comment by ettaboyle** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
